assets/components/src/select-control/style.scss

```diff
@@ -2,114 +2,29 @@
  * Select Control
  */
 
-@import '../../../shared/scss/colors';
 @import '~@wordpress/base-styles/colors';
 
-.newspack-select-control {
-	color: $gray-700;
-	font-size: 14px;
-	line-height: 24px;
-	margin: 32px 0;
+.components-select-control {
+	&__input {
+		max-width: 100% !important;
 
-	.components-base-control__field {
-		margin: 0;
-	}
-	.components-input-control__label,
-	.components-base-control__label {
-		color: $gray-900 !important;
-		display: flex;
-		font-size: 14px;
-		font-weight: bold;
-		line-height: 24px;
-		padding-bottom: 0 !important;
-		margin-bottom: 0;
-	}
-
-	.components-select-control {
-		border-radius: 3px;
-		font-size: 14px;
-		line-height: 24px;
-
-		div.components-input-control__backdrop {
-			border-color: $gray-300;
-		}
-
-		select.components-select-control__input {
-			color: $gray-700;
-			border-color: $gray-300;
-			font-size: 16px;
-			height: 48px;
-			line-height: 24px;
-			max-width: 100%;
-			min-height: 48px;
+		&[multiple] {
```

Comment:
The selected items in a multiselect component stop just short of going full-width, and look somewhat strange. I think we may also want to add some similar padding so that the top of the first selection aligns with the text in the single-select component next to it:

<img width="701" alt="Screen Shot 2022-06-09 at 1 22 50 PM" src="https://user-images.githubusercontent.com/2230142/172928030-1cdd2da2-4985-4f73-9bb2-253d708a3908.png">

Reply:
Fixed: https://github.com/Automattic/newspack-plugin/pull/1676/commits/8d757e8f66bc04c39db290e2212e30b7ac5a527c

Tbh, I'm don't think we even use a multiple select other than the demo page.